packages/firestore/lite/index.d.ts

```diff
@@ -0,0 +1,338 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '@firebase/app-types';
+
+export type DocumentData = { [field: string]: any };
```

Comment:
Should this be an interface vs a type?

Reply:
This is mainly because it matches the type in the current API (https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore-types/index.d.ts#L20)

AFAIK there is no real difference, but this particular type should never be used in a class hierarchy.

The GitHub Actions seems to use different Lint settings from what we do. I only get lint violations for the usage of `any`.

Reply:
As a separate work item we should make sure our local development and GHA match up.

Reply:
@Feiyang1

Reply:
FWIW, this is the first time I see this kind of discrepancy (but I am also not sure when we started Lint checking in this Rep, the nodejs-firestore repo has had this check on for a while).